pkg/storage/calico/bgpConfiguration_storage.go

```diff
@@ -76,25 +76,25 @@ type BGPConfigurationConverter struct {
 
 func (gc BGPConfigurationConverter) convertToLibcalico(aapiObj runtime.Object) resourceObject {
```

Comment:
I think that these functions are totally useless now, since the input type is the same as the output type. We should just remove them.